src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -69,6 +73,23 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 });
 
+            services.AddHealthChecks()
+                .AddCheck(
+                    name: "self",
+                    check: () => HealthCheckResult.Healthy(),
+                    tags: new[] { HealthCheckTags.Live })
+                .AddSmtpHealthCheck(
+                    setup: (smtp) =>
+                    {
+                        smtp.Host = smtpSettings.Host;
+                        smtp.Port = smtpSettings.Port;
+                        smtp.ConnectionType = SmtpConnectionType.TLS;
+                    },
+                    name: "smtp",
+                    failureStatus: HealthStatus.Unhealthy,
+                    tags: new[] { HealthCheckTags.Ready },
+                    timeout: TimeSpan.FromSeconds(10));
```

Comment:
As above, I'm not sure specifying the argument names is adding much here. Personally, I prefer it without them but see what you think.

Reply:
👍